### PR TITLE
Release `0.12.0`

### PR DESCRIPTION
**Pull request by nleroy917**
### TODO:
- [x] bump version
- [x] changelog?

This fixes **a ton** of issues with the sample table and overall project page.

#### Related issues:
- #350 
- #349 
- #347 
- #346 
- #345 
- #344 
- #343 
- #342 
- #340 
- #339 
- #338 
- #337

### CODE REVIEW

File: pephub/_version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "0.11.8"
+__version__ = "0.11.9"
```
Comment (khoroshevskyi) [inline pephub/_version.py]: I would change version to 0.12 because we added history

**Review by khoroshevskyi — APPROVED** [review]
lgtm

**Comment by sanghoonio** [comment]
On dev, seems like PEPs aren't populating in project lists
image

**Comment by nleroy917** [comment]
Its possible that this is a result of the fact that the `dev` client deploys the latest updates... but it doesn't include updates to the API. The dev client currently reflects changes in the API that will be deployed on release

File: pephub/_version.py (hunk 1 above)
Comment (nleroy917) [reply pephub/_version.py]: Done

**Pull request merged**